src/v/config/configuration.h

```diff
@@ -321,6 +319,8 @@ struct configuration final : public config_store {
     // cloud storage read and write enabled
     property<std::optional<size_t>> retention_local_target_bytes_default;
     property<std::chrono::milliseconds> retention_local_target_ms_default;
+    property<std::optional<uint64_t>> retention_local_target_capacity_bytes;
```

Comment:
how does it work during an upgrade?

Reply:
not enabled

Reply:
sorry i meant: how this property rename will be handled during an upgrade from v23.1 to v23.2? but I see in the docs that log_storage_max_usage_interval and log_storage_target_size were new to v23.2 anyway